pyxem/utils/diffraction.py

```diff
@@ -26,10 +26,12 @@
 import pyxem as pxm  # for ElectronDiffraction2D
 
 from scipy.interpolate import interp1d
+from scipy.signal import fftconvolve
 from skimage import transform as tf
 from skimage.feature import match_template
 from skimage import morphology, filters
 from skimage.draw import ellipse_perimeter
+from skimage._shared.utils import _supported_float_type
```

Comment:
We can just copy it.  I had that there because I was trying to think if anyone else would be interested in this method/ should I try to upstream it.   I don't really think there are terribly many places this might be relevent so I decided against it.